.github/workflows/060_generate_benchmark_weights.yml

```diff
@@ -0,0 +1,67 @@
+name: Generate benchmark weights
+
+on:
+  workflow_dispatch:
+
+jobs:
+  generate-benchmark-weights:
+    runs-on: [self-hosted, tfchainrunner01]
+    container:
+      image: threefolddev/tfchain:4
+      env:
+        DEBIAN_FRONTEND: noninteractive
+        PATH: /root/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin
+        RUSTUP_HOME: /root/.rustup
+        CARGO_HOME: /root/.cargo
+
+    steps:
+      - name: Fail if branch is main
+        if: github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/development'
+        run: |
+          echo "This workflow should not be triggered with workflow_dispatch on main branch"
+          exit 1
+
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Chown user
+        run: |
+          chown -R $USER:$USER $GITHUB_WORKSPACE
+
+      - name: Cache build
+        uses: actions/cache@v3
+        timeout-minutes: 6
+        continue-on-error: true
+        with:
+          path: |
+            /root/.cargo/bin/
+            /root/.cargo/registry/index/
+            /root/.cargo/registry/cache/
+            /root/.cargo/git/db/
+            substrate-node/target/
+          key: ${{ runner.os }}-tfchain-build-cache-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: ${{ runner.os }}-tfchain-build-cache
+
+      - name: Build
+        run: |
+          cd substrate-node
+          cargo build --profile=bench --features runtime-benchmarks
+
+      - name: Run benchmarking
+        run: |
+          cd substrate-node
+          for weights_rs_file in ./pallets/*/src/weights.rs
+          do
+            rm $weights_rs_file
+            pal_name=$(awk -F'pallets/|/src' '{print $2}' <<< $weights_rs_file)
+            ./target/release/tfchain benchmark pallet --chain=dev --pallet="$pal_name" \
+            --extrinsic="*" --steps=50 --repeat=20 --execution=wasm --heap-pages=409 \
```

Comment:
`--execution` is this benchmark option still valid on 2.7?
I'm getting this error when try it locally
```
error: unexpected argument '--execution' found
tip: a similar argument exists: '--wasm-execution'
```

Reply:
Good catch
This option seems to be deprecated